File: sdks/thunkmetrc-py/wrapper/src/thunkmetrc/wrapper/models/create_adjust_plant_batches_request.py
from typing import TypedDict, List, Optional, Any, Generic, TypeVar, TYPE_CHECKING
from typing_extensions import TypedDict

if TYPE_CHECKING:
    from . import *

class CreateAdjustPlantBatchesRequest(TypedDict, total=False):
    AdjustmentDate: str
    AdjustmentReason: str
    PlantBatchName: str
    Quantity: int
    ReasonNote: str

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/CreateAdjustPlantBatchesRequest.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class CreateAdjustPlantBatchesRequest(
    @JsonProperty("AdjustmentDate")
    val adjustmentDate: String? = null,
    @JsonProperty("AdjustmentReason")
    val adjustmentReason: String? = null,
    @JsonProperty("PlantBatchName")
    val plantBatchName: String? = null,
    @JsonProperty("Quantity")
    val quantity: Int? = null,
    @JsonProperty("ReasonNote")
    val reasonNote: String? = null
)
